src/components/delegate/delegate.html

```diff
@@ -26,8 +26,7 @@ <h1>Delegate</h1>
 							<tr>
 								<td class="left-padding-xs left-padding-s left-padding-m left-padding-l double"><strong>Name</strong></td>
 								<td class="right-padding-xs right-padding-s right-padding-m right-padding-l double text-right">
-									<a class="glyphicon glyphicon-user" data-ng-if="vm.address.delegate.username | proposal:$root.delegateProposals" data-ng-href="https://forum.lisk.io/viewtopic.php?f=48&t={{vm.address.delegate.username | proposal:$root.delegateProposals: 'topic'}}" title="{{vm.address.delegate.username | proposal:$root.delegateProposals: 'description'}}" target="_blank"></a>
-									<a href="/delegate/{{vm.address.delegate.address}}">{{vm.address.delegate.username}}</a>
+									<a class="glyphicon glyphicon-user" href="/delegate/{{vm.address.delegate.address}}">{{vm.address.delegate.username}}</a>
```

Comment:
This should be `<a href="/delegate/{{vm.address.delegate.address}}">{{vm.address.delegate.username}}</a>`